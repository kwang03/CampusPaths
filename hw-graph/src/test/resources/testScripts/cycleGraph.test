#Tests the behavior of the graph with a cycle

#Create graph
CreateGraph myGraph

#Adds some nodes to the graph
AddNode myGraph n1
AddNode myGraph n2
AddNode myGraph n3
AddNode myGraph n4
AddNode myGraph n5

#Adds edges between nodes to create a cycle
AddEdge myGraph n2 n3 e23
AddEdge myGraph n3 n5 e35
AddEdge myGraph n5 n4 e54
AddEdge myGraph n4 n1 e41
AddEdge myGraph n1 n2 e12

#Lists the children of nodes
ListChildren myGraph n1
ListChildren myGraph n2
ListChildren myGraph n3
ListChildren myGraph n4
ListChildren myGraph n5