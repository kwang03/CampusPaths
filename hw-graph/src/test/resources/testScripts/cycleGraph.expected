#Tests the behavior of the graph with a cycle

#Create graph
created graph graphName

#Adds some nodes to the graph
added node n1 to myGraph
added node n2 to myGraph
added node n3 to myGraph
added node n4 to myGraph
added node n5 to myGraph

#Adds edges between nodes to create a cycle
added edge e23 from n2 to n3 in myGraph
added edge e35 from n3 to n5 in myGraph
added edge e54 from n5 to n4 in myGraph
added edge e41 from n4 to n1 in myGraph
added edge e12 from n1 to n2 in myGraph

#Lists the children of nodes
the children of n1 in myGraph are: n2(e12)
the children of n2 in myGraph are: n3(e23)
the children of n3 in myGraph are: n5(e35)
the children of n4 in myGraph are: n1(e41)
the children of n5 in myGraph are: n4(e54)